shared/types.ts

```diff
@@ -38,3 +38,116 @@ export type CreateSession = z.infer<typeof createSessionSchema>;
 export type Feedback = z.infer<typeof feedbackSchema>;
 
 export type CreateFeedback = z.infer<typeof createFeedbackSchema>;
+
+export type KeyboardEventKeys =
+  | ' ' // Space
+  | '!'
+  | '@'
+  | '#'
+  | '$'
+  | '%'
+  | '^'
+  | '&'
+  | '*'
+  | '('
+  | ')'
+  | '_'
+  | '+'
+  | '-'
+  | '='
+  | '0'
+  | '1'
+  | '2'
+  | '3'
+  | '4'
+  | '5'
+  | '6'
+  | '7'
+  | '8'
+  | '9'
+  | 'a'
+  | 'b'
+  | 'c'
+  | 'd'
+  | 'e'
+  | 'f'
+  | 'g'
+  | 'h'
+  | 'i'
+  | 'j'
+  | 'k'
+  | 'l'
+  | 'm'
+  | 'n'
+  | 'o'
+  | 'p'
+  | 'q'
+  | 'r'
+  | 's'
+  | 't'
+  | 'u'
+  | 'v'
+  | 'w'
+  | 'x'
+  | 'y'
+  | 'z'
+  | 'A'
+  | 'B'
+  | 'C'
+  | 'D'
+  | 'E'
+  | 'F'
+  | 'G'
+  | 'H'
+  | 'I'
+  | 'J'
+  | 'K'
+  | 'L'
+  | 'M'
+  | 'N'
+  | 'O'
+  | 'P'
+  | 'Q'
+  | 'R'
+  | 'S'
+  | 'T'
+  | 'U'
+  | 'V'
+  | 'W'
+  | 'X'
+  | 'Y'
+  | 'Z'
+  | 'Backspace'
+  | 'Tab'
+  | 'Enter'
+  | 'Shift'
+  | 'Control'
+  | 'Alt'
+  | 'Pause/Break'
+  | 'Caps Lock'
+  | 'Escape'
+  | 'Space'
+  | 'Page Up'
+  | 'Page Down'
+  | 'End'
+  | 'Home'
+  | 'ArrowLeft'
+  | 'ArrowUp'
+  | 'ArrowRight'
+  | 'ArrowDown'
+  | 'Insert'
+  | 'Delete'
+  | 'F1'
+  | 'F2'
+  | 'F3'
+  | 'F4'
+  | 'F5'
+  | 'F6'
+  | 'F7'
+  | 'F8'
+  | 'F9'
+  | 'F10'
+  | 'F11'
+  | 'F12'
+  | 'Num Lock'
+  | 'Scroll Lock';
```

Comment:
Onko tämä turha? 

Pyysin AI:ta tekemään `type`n, jossa olisi kaikki yleisemmin käytetyt keyboard eventit

Jos tuo on ihan ok olla, onko `shared/types.ts` hyvä paikka?

Reply:
Ihan näppärä tyyppi, ja tuo `shared/types.ts` on hyvä paikka.

Reply:
Olisiko hyvä tehdä vaikka `/utils/`-kansion alle oma `types`-tiedosto, joka sisältäisi vain frontin typet (eli tässä tapauksessa keyboard eventit)? Vai tuoko vaan turhia kansiota ja näin ollen hankaloittaa